CMakeLists.txt

```diff
@@ -344,7 +349,7 @@ install(TARGETS darklib EXPORT DarknetTargets
   RUNTIME DESTINATION "${INSTALL_BIN_DIR}"
   LIBRARY DESTINATION "${INSTALL_LIB_DIR}"
   ARCHIVE DESTINATION "${INSTALL_LIB_DIR}"
-  PUBLIC_HEADER DESTINATION "${INSTALL_INCLUDE_DIR}"
+  PUBLIC_HEADER DESTINATION "${INSTALL_INCLUDE_DIR}/darknet"
```

Comment:
this is the main change of this PR

Reply:
@cenit 

> Discussing about publishing the tool and live in peace with other libraries, I was informed that we export some header files that have the same name as other libraries. The best solution is to wrap them inside a darknet folder. 

Should I change anything in the source code or move some headers?
Or is changes in your PR enough?

Reply:
changes in this PR should be enough to be compliant. And I tried my best to not touch the code in any way, so they should be transparent for you. Please confirm that everything still works as expected for you, then I will ask if these changes are ok also for package maintainers